chainer/training/extensions/util.py

```diff
@@ -1,4 +1,7 @@
+import datetime
```

Comment:
`IteratorProgressBar` is used from outside `extensions`.
How about moving `IteratorProgressBar` to the upper level, in e.g. `chainer/training/_progress_bar.py`, and moving the existing objects (`set_console_cursor_position`, `erase_console`, etc.) in e.g. `chainer/training/_terminal.py`?

Reply:
> IteratorProgressBar is used from outside extensions.

Where are you pointing out?

Reply:
Oh, sorry, it's my misunderstanding.
Please ignore this comment.